transfer-data/destinations.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: 'Destinations'
+description: 'This page explains how to set up destinations.'
+---
+
+<Note>
+Flow is currently in preview. To try it out, [sign up for a free preview](https://forms.axiom.co/flow).
```

Comment:
"preview [...] preview."

To try it out, sign up for a free account? 

I would also prefer that we link to the Flow page in-app. We should ask for an org-less route like others in Axiom, i.e. https://app.axiom.co/flows, which redirects to https://app.axiom.co/<org-id>/flows. We use this in docs for many other routes.

Reply:
Good idea about the org-less route. Asking here: https://watchlyhq.slack.com/archives/C1NJU9Y6B/p1723133547478749

Reply:
Implemented in https://github.com/axiomhq/app/pull/2642

Reply:
This is now live.